src/show/pages/public/CreateProject/Subviews/index.js

```diff
@@ -0,0 +1,443 @@
+      </div>
+    </div>
+  );
+};
+
+const TeamMembersStep = ({ nextStep, previousStep, project, setProject }) => {
+  return (
+    <div className='mt-12 px-20'>
+      <div className='text-center text-3xl text-gray-500'>
+        Enter your project members
+      </div>
+      <div className='flex w-full mt-10'>
+        <div className='flex w-1/2 h-full'>
+          <div className='w-3/4 px-2'>
+            {project.team_members.map((member, index) => (
+              <TextField
+                key={index}
+                value={member.name}
+                className='rounded-2xl mt-2 items-center text-center'
+                onChange={e => {
+                  let newTeamMembers = project.team_members;
+                  newTeamMembers[index] = { ...member, name: e.target.value };
+                  setProject({ ...project, team_members: newTeamMembers });
+                }}
+              />
+            ))}
+          </div>
+        </div>
+        <div className='flex w-1/2 h-full justify-end'>
+          <div className='w-full px-2'>
+            {project.team_members.map((member, index) => (
+              <TextField
+                key={index}
+                value={member.avatar_url}
+                className='rounded-2xl mt-2 items-center text-center'
+                onChange={e => {
+                  let newTeamMembers = project.team_members;
+                  newTeamMembers[index] = {
+                    ...member,
+                    avatar_url: e.target.value
+                  };
+                  setProject({ ...project, team_members: newTeamMembers });
+                }}
+              />
+            ))}
+          </div>
+        </div>
+      </div>
+
+      <div className='flex mt-20 justify-between cursor-pointer'>
+        <div
+          onClick={() => previousStep()}
+          className='flex items-center hover:scale-110 transition-all duration-300'>
+          <BsArrowBarLeft size='2rem' />
+          Back
+        </div>
+        <div
+          onClick={() => {
+            nextStep();
+          }}
+          className='flex items-center cursor-pointer hover:scale-110 transition-all duration-300'>
+          Next
+          <BsArrowBarRight size='2rem' />
+        </div>
+      </div>
+      <div className='mt-20 px-60'>
+        <ProgressBar percentage={90} color='bg-primary-600' />
+      </div>
+    </div>
+  );
+};
+
+const PublishStep = ({ nextStep, postData, previousStep }) => {
+  return (
+    <div className='mt-18 px-20'>
+      <div className='mt-60'>
+        <div className='flex justify-center text-center text-3xl text-gray-500'>
+          Ready to publish?
+        </div>
+        <div className='flex justify-center'>
+          <div className='flex justify-center h-full mt-2 w-40  text-2xl rounded-full'>
+            <Button
+              onClick={() => {
+                nextStep();
+                postData();
+              }}
```

Comment:
```suggestion
              onClick={(e) => postData(e)}
```

Reply:
There's no next step here. It's the final step already. Delete nextStep() and make it in one line. Also you are supposed to be getting and passing and event (`e`).

Reply:
why do i get and pass an event?